pypesto/objective/petab_import.py

```diff
@@ -621,6 +630,66 @@ def _find_model_name(output_folder):
     return os.path.split(os.path.normpath(output_folder))[-1]
 
 
+def _merge_preeq_and_sim_pars(parameter_mappings, scale_mappings):
```

Comment:
Could be implemented in PEtab, but okay for now.

Reply:
Do we have a policy on type hints in pyPESTO? Should be formulated at some point...

Reply:
yes, so far we have "ok" type hints often in the docstring, but these do not have to be exact. we should use type hints more, definitely.